src/Build/NuGet.Build.Packaging.Tasks/CreatePackage.cs

```diff
@@ -140,12 +153,12 @@ select item.GetNuGetTargetFramework()))
 					definedDependencyGroups.Add(targetFramework.GetFrameworkString(),
 												new PackageDependencyGroup(targetFramework, Array.Empty<PackageDependency>()));
 
-			manifest.Metadata.DependencyGroups = definedDependencyGroups.Values;
+			manifest.Metadata.DependencyGroups = definedDependencyGroups.Values.Concat(manifest.Metadata.DependencyGroups);
```

Comment:
II think what would be needed is more of a merge here, than a concat, since the concat would end up with duplicate groups? Is that supported in the nuspec?